scripts/generateColorLess.js

```diff
@@ -1,92 +1,16 @@
 #!/usr/bin/env node
-
-const fs = require('fs');
 const path = require('path');
-const glob = require('glob');
-const postcss = require('postcss');
-const less = require('less');
-
-const COLOR_MAP = {
-  '#e6f7ff': 'color(~`colorPalette("@{primary-color}", 1)`)', // @primary-1
-  '#bae7ff': 'color(~`colorPalette("@{primary-color}", 2)`)', // @primary-2
-  '#40a9ff': 'color(~`colorPalette("@{primary-color}", 5)`)', // @primary-5
-  '#1890ff': '@primary-color',
-  '#096dd9': 'color(~`colorPalette("@{primary-color}", 7)`)', // @primary-7
-  '#46a6ff': 'tint(@primary-color, 20%)',
-  '#8cc8ff': 'tint(@primary-color, 50%)',
-  'rgba\\(24, 144, 255, 0.2\\)': 'fade(@primary-color, 20%)',
+const { generateTheme } = require('antd-theme-generator');
+
+const options = {
+  stylesDir: path.join(__dirname, '../site/theme/static'),
+  antDir: path.join(__dirname, '../node_modules/antd'),
+  varFile: path.join(__dirname, '../node_modules/antd/lib/style/themes/default.less'),
```

Comment:
Same here, can we use `path.join(__dirname, '../components/style/themes/default.less')`